config/kernel-timer.m4

```diff
@@ -1,32 +1,71 @@
 dnl #
-dnl # 4.15 API change
-dnl # https://lkml.org/lkml/2017/11/25/90
-dnl # Check if timer_list.func get passed a timer_list or an unsigned long
-dnl # (older kernels).  Also sanity check the from_timer() and timer_setup()
-dnl # macros are available as well, since they will be used in the same newer
-dnl # kernels that support the new timer_list.func signature.
```

Comment:
It looks to me this shouldn't be removed as the `timer_list.func` compile time test still exists.